contracts/message-bus/MessageOutbox.sol

```diff
@@ -72,13 +72,17 @@ contract MessageOutbox is MessageBox {
     /**
      * @notice Setup message outbox.
      *
-     * @dev Function requires:
-     *          - outboundChannelIdentifier must be zero
-     *          - metachainId must not be zero
-     *          - messageInbox address must not be zero
-     *
      * @param _metachainId Metachain identifier.
      * @param _messageInbox MessageInbox contract address.
+     *
+     * \pre The function can be called only once.
+     * \pre `_metachainId` must not be zero.
+     * \pre `_messageInbox` address must not be zero.
+     *
+     * \post Sets `messageInbox` storage variable with `_messageInbox`.
+     * \post Sets `outboundChannelIdentifier` storage variable.
```

Comment:
Same question wrt specifying (mentioning, making clear) the input parameters. Do we need them or their are obvious?

Reply:
Same as above